pages/docs/data-ops/resource-lineage.mdx

```diff
@@ -0,0 +1,15 @@
+# Resource Lineage
+
+You can use the resource lineage view to see your resources such as models, metrics, explorations, and dashboards in a dependency graph or DAG.  There are two ways to access the Resource Lineage graph:
+
+### Via the DataOps Page
+
+1. Click on your project name in the `nav bar`.
+2. Select `DataOps` from the menu.
+3. Go to `Resource Lineage` or follow the link: [Resource Lineage](https://glean.io/app/p/data-ops#resource_lineage).
+
+### On a Per-Resource Basis
+
+1. Find the resource you want to view in the Resource Lineage graph.
+2. Click on the dropdown three dots button at the top of any resource page.
```

Comment:
```suggestion
2. Click on the dropdown next to a resource's name at the top of any Exploration, or the three-dot dropdown at the top of a Dashboard.
```

Reply:
This actually won't be right after we launch changes to saved explorations.  Will hold off on merging until that is live.